internal/data-importer/deploy/psmdb/dataimporter.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: everest.percona.com/v1alpha1
+kind: DataImporter
+metadata:
+  name: everest-percona-psmdb-operator-data-importer
+spec:
+  displayName: "Percona PSMDB Operator Data Importer"
+  description: |
+    Data importer for importing logical backups taken using the Percona PSMDB Operator.
+  supportedEngines:
+  - psmdb
+  jobSpec:
+    image: "asia-south1-docker.pkg.dev/percona-everest/mayank-everest-test/everest-operator"
```

Comment:
as discussed in the pxc CR, it should point to the main `everest-operator` image

Reply:
done